boa/src/builtins/object/mod.rs

```diff
@@ -109,6 +121,110 @@ impl Object {
         Ok(obj)
     }
 
+    /// `Object.getOwnPropertyDescriptor( object, property )`
+    ///
+    /// Returns an object describing the configuration of a specific property on a given object.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://www.ecma-international.org/ecma-262/10.0/index.html#sec-object.getownpropertydescriptor
```

Comment:
Could we link the rolling spec instead of one specific version?
https://tc39.es/ecma262/#sec-object.getownpropertydescriptor